### PR TITLE
improve docker image size

### PR DESCRIPTION
**Pull request by fblackburn1**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -14,17 +28,13 @@ RUN true \
...
-WORKDIR /usr/src/wazo-setupd
-RUN true \
-  && pip install -r /usr/src/wazo-setupd/requirements.txt \
-  && python setup.py install \
-  && cp -r etc/* /etc
-
 EXPOSE 9302
 
-CMD ["python3", "-u", "/usr/local/bin/wazo-setupd"]
+ENV PYTHONUNBUFFERED=TRUE
```
Comment (bloom1) [inline Dockerfile:36]: Why is this necessary?

Comment (fblackburn1) [reply Dockerfile:36]: I personally, never experiment this behavior but it's to avoid to hide some log by the buffer when a crash occurs. I think seb has already debug this and it take some time to understand why... so it's why we have this option activated in some services
